client/src/components/CRM/Reporting/TransactionSummaryReport/OrderLevelFeesByPaymentComponent.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {DataGrid, GridColDef, GridValueGetterParams} from '@mui/x-data-grid';
+
+const OrderLevelFeesByPaymentComponent = () => {
```

Comment:
Great job on creating the mock data as json objects & not hard coding your data into your components. this will save the API developers a lot of time when determining the content to return from their SQL queries.